code/modules/mob/living/silicon/ai/ai_say.dm

```diff
@@ -129,7 +129,7 @@
 
 	log_message("made a vocal announcement with the following message: [message].", LOG_GAME)
 	log_talk(message, LOG_SAY, tag="VOX Announcement")
-	say(";[message]", forced = "VOX Announcement")
+	minor_announce(capitalize(message), "[name] VOX Announces:", sound_override = TRUE)
```

Comment:
I don't think this should mention the VOX--how else is the AI speaking in character?

Reply:
```suggestion
	minor_announce(capitalize(message), "[name] announces:", sound_override = TRUE)
```

Reply:
I thought there should be a distinction from the normal comms console announcement

Reply:
I see, I think it's a little weird because I don't know what the difference would possibly be IC. Not a blocking concern though

CC @EOBGames if you have any thoughts on this